container_pipeline/workers/linter.py

```diff
@@ -80,30 +80,29 @@ def lint(self):
         try:
             out, err = run_cmd_out_err(command)
             if err == "":
-                response = self.handle_lint_success(out)
                 self.logger.info(
-                    "Docker file linting successful going for job creation")
+                    "Dockerfile linting successful going for openshift job creation")
+                response = self.handle_lint_success(out)
                 if not response["job_created"]:
                     self.logger.warning(
-                        "Openshift project is not created putting it back to tube")
+                        "Openshift project is not created putting it back to linter tube")
```

Comment:
`"not created putting it"` putting what? Please do `s/it/job` or something similar.